src/Ombi.Helpers/StringHelper.cs

```diff
@@ -131,7 +131,7 @@ public static string StripCharacters(this string str, params char[] chars)
 
         public static string ToHttpsUrl(this string currentUrl)
         {
-            return currentUrl.Replace("http://", "https://");
+            return currentUrl?.Replace("http://", "https://");
```

Comment:
Nice work! Looks good, 

Can we please add a unit test for this where the `currentlyUrl` is null?

Reply:
Done. =)